### PR TITLE
Header added

### PR DESCRIPTION
**Pull request by EroAuditore**
Header with font awesome added

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,11 +7,17 @@
...
+    <header>
+      <nav class="navbar navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">
+            <i class="fab fa-product-hunt d-inline-block"> </i>
+            Pokescripts
+          </a>
+        </div>
+      </nav>
+    </header>
```
Comment (initseis) [inline src/index.html]: I think is a good idea to add some options to the navbar like the example.

example

**Review by initseis — COMMENTED** [review]
I think the linter error that you have is for using double quotes It is simple to resolve.
Please check my comment below.

File: src/index.html (hunk 1 above)
Comment (EroAuditore) [reply src/index.html]: Ready you can check this

Comment (initseis) [reply src/index.html]: Well done!! 👍🏽👍🏽
You can merge it.

**Pull request merged**